src/nodejs/analytics/analytics.cpp

```diff
@@ -133,9 +133,17 @@ v8::Local<v8::Value> TNodeJsAnalytics::TNMFTask::WrapResult() {
 
 TNodeJsSvmModel::TNodeJsSvmModel(const PJsonVal& ParamVal):
         Algorithm("SGD"),
+        Kernel("LINEAR"),
```

Comment:
Can we reuse `TParams` and `TModel` classes here to not have another separate set of variables for same parameters?